### DISCUSSION TITLE
Xoptfoil2 for DIY Paragliders & Ultralights

### DISCUSSION THREAD

**Discussion by goney3** [jxjo/Xoptfoil2#5] [General]
### Brought over from the Introductions thread:
 
_Questions:_

_- Does Xoptfoil2 allow for creating closed trailing edge airfoils? I ask as this is a requirement in DIY paraglider designs (the trailing edge is sewn closed)._
_- Will Xoptfoil2 be able to handle "single skin" airfoil designs? (leading edge and top surface without a bottom). This is an up-and-coming area of DIY paragliding that allows for lighter wings and less packing volume to create wings that can fit in extremely small spaces (great for traveling!)._
_- Will Xoptfoil2 have the ability to optimize airfoils on a "mush stall characteristic" as an option? I know the last version was more focused on Cl, Cd, and Cm. Our current design does about Cl-max 1.8 with compromise on Cd and Cm to have a jack-of-all-trades airfoil for USA Ultralights. We're looking forward to creating new freely-available airfoils to better serve the average DIY pilot with higher safety._

### Replying to jxjo's answers:

Thank you for letting me know that single skin is not possible with Xfoil (since they don't work in XFLR5). I have been putting off learning CFD to study these shapes (as something like OpenFOAM seems very complicated), maybe one day it'll be easier to simulate single skins. 

We've been comparing known airfoils and their stall conditions to various results. We're not searching for "perfect accuracy" but better airfoils than what is currently available to builders. I am looking forward to field verifying our current design with a ground handling wing.

**Comment by jxjo** [comment]
Hello Keith,

... I made just a quick try with quite a thin airfoil which may be comparable to a single skin in its aerodynamic properties ...

![grafik](https://github.com/jxjo/Xoptfoil2/assets/58261241/80dbb43f-d6d4-43fb-8bcd-ef2069b63a24)

Amazingly Xflr5 (xfoil) can calculate it (not knowing how close the results are in 'reality') 

![grafik](https://github.com/jxjo/Xoptfoil2/assets/58261241/3e754f25-376f-412f-8dd0-ec410202f0b4)

The airfoil was created with the [AirfoilEditor ](https://github.com/jxjo/PlanformCreator2 )using Bezier curves (quite similar to the Spline airfoil in Xflr5) ... Maybe you want to test this a little more ...

regards 

Jochen

**Comment by goney3** [comment]
Pere Casellas made some nice drawings of single skin profiles here: http://www.laboratoridenvol.com/projects/skd/skdevelopment.en.html

I didn't have much luck trying to create something like it with the AirfoilEditor, but that is a great program you made!

**Reply by jxjo** [reply]
Hello Keith,
... thanks for the drawings - and oh! 
I hadn't thought, that also the leading edge is thin (open) - amazing what is flying ... ;-) 
Imho Xfoil is only able to calculate something like a rounded leading edge therefore such a skin airfoil can only be be approximated.

**Comment by goney3** [comment]
I had Professor Selig respond to my question I sent him last year, I am stunned he acknowledged the question! He took the Ultralight Airplane Workshops airfoil and ran an analysis on it and found that it wasn't very smooth. Not knowing how to do that in XFLR5 I took your PlanformCreator2 and ran it through the Bezier tool... really neat program! The matching option is AWESOME! ... does this smooth the airfoil when ran? I am only assuming at this point. 
[UAK02-180-31-39-286.txt](https://github.com/jxjo/Xoptfoil2/files/14842064/UAK02-180-31-39-286.txt)
[UAK02-repan-norm-bezier.txt](https://github.com/jxjo/Xoptfoil2/files/14842065/UAK02-repan-norm-bezier.txt)

**Reply by jxjo** [reply]
Hello Keith, 
... glad you like the Airfoil Editor!
What do you mean with "smoothing"?

**Reply by goney3** [reply]
From Professor Selig:

_"I ran your airfoil in XFOIL and then looked the curvature.  I took some screen shots and attached them._

_I like to see curvature looking relatively smooth.  When that happens the velocity distribution will be smooth.  For airfoils like that, then XFOIL predictions are going to be about a reliable as they can be (that does not mean XFOIL matches experiment, however).  If the velocity distributions are not smooth, then I don't trust XFOIL and by extension I would not trust XFLR5._

_Your airfoil is not smooth.  You can see the non-smooth Cp (~velocity distribution) in the attached.  The curvature plots out of Rhino are showing the same thing.  Then in XFOIL, it is showing the same thing._

_I think if you want to know XFOIL results for your airfoil, then take your airfoil into CAD and fit it w/ a spline curve and get a new airfoil.  Break that smooth curve into points.  Check that the points are smooth (smooth curvature, or at least without jaggies).  Then run it in XFOIL and check it out there.  I would not use XFLR5 when going directly to the source code / program and so much easier._

_As for how this all applies to your real paraglider:  If you want to get a good assessment out of XFOIL that you can trust, it matters a lot.  If you glider flies how you want it, then none of this matters._

_All that said -- know this: XFOIL only gives a prediction and the prediction does not match any experiment perfectly._

_Regards,
Michael_
"

![IMG_3017](https://github.com/jxjo/Xoptfoil2/assets/19561317/b0d1c518-6b6f-4063-b5ef-4dc4cb52a5ce)
![IMG_3018](https://github.com/jxjo/Xoptfoil2/assets/19561317/953aa6f4-9d8e-4025-8d27-8ffebb032880)
![IMG_3019](https://github.com/jxjo/Xoptfoil2/assets/19561317/665a3328-935f-412a-a4b6-326cd9f34baf)
![IMG_3020](https://github.com/jxjo/Xoptfoil2/assets/19561317/bb05e55c-c49e-4e2c-a806-c345714ffcc2)

**Reply by goney3** [reply]
Here is my efforts that I have been trying to work on from the perspective of Airfoil Editor:
![UAK02-Curvature](https://github.com/jxjo/Xoptfoil2/assets/19561317/0c0a4c7d-ef07-4688-9b53-a4e7547cc75b)
This one shows our original airfoil and what appears to be noise in the curvature.

![UAK02-Hanning-Curvature](https://github.com/jxjo/Xoptfoil2/assets/19561317/ee3eca4a-adc3-4237-9838-62b958fabda0)
Here I took the XFLR5 Inverse module and ran the standard Hanning filter to smooth the airfoil, the result is less jittery but is still lumpy.

![UAK02-Bezier-Bezier-Curvature](https://github.com/jxjo/Xoptfoil2/assets/19561317/26a7b865-1798-418b-a270-19e04e999d60)
I ran your bezier tool twice and it appears to be a lot more smooth compared to the original. Though this did change the airfoils shape slightly.

**UPDATE:** I just heard back from Professor Selig just now, he approved of the bezier version!
![xfoil-uak02-bezier-Selig](https://github.com/jxjo/Xoptfoil2/assets/19561317/916559d8-e824-4f28-b664-d7a50dc94fed)

He also recommended a program call AFSMO from the 1980's for smoothing, but said your program worked just as well: 
https://www.rcgroups.com/forums/showthread.php?4166567-Airfoil-smoothing-repair-with-NASA-s-AFSMO-code

He also mentioned: _"Going back to your prior version/airfoil, I think the question raised by non-smooth coordinates is that those velocity wiggles can cause transition to be predicted sooner than in real-life after the airfoil might be smoothed out naturally from fabrication.  So XFOIL in that case is not going to be reliable generally.  Depending on how it affects transition, XFOIL can be very wrong w/ non-smooth coordinates."_

Apparently he also maintains PROFOIL on Github: https://github.com/m-selig/PROFOIL

**Comment by jxjo** [comment]
Hello Keith, 
... ah - you really meant 'smoothing' ... ;-) 
The results really look good. How many Bezier control points did you use? 
I don't know, if you already have seen it: I'm currently working on a little docu for Xoptfoil2 and wrote some [remarks about 'curvature' ](https://jxjo.github.io/Xoptfoil2/docs/geometry#curvature-of-an-airfoil). Maybe you'll find it useful.

regards
Jochen

**Reply by goney3** [reply]
I used 8 points since that's the maximum your program allowed for the auto matching. Though I think it only was able to match 6 points. Thank you for the manual link, I'll definitely check it out, thank you again! :)
